swagger-apis/enrollments/1.0.0-beta.1.yml

```diff
@@ -138,7 +142,7 @@ paths:
         content:
           application/jwt:
             schema:
-              allOf:
+              allOf: #Não seria o oneOf ou o anyOf? Salvo engano, oneOf parece ser mais adequado, porque precisariamos de pelo menos um elemento em comum entre os schemas. allOf exige que todos os elementos tenham propriedades em comum.
```

Comment:
agreed

Reply:
O objetivo no uso de allOf é a composição dos tipos internos em um único objeto em acordo com a especificação: https://swagger.io/docs/specification/data-models/oneof-anyof-allof-not/#allof
Desta maneira, os subtipos internos podem ser reusados. 

O uso de oneOf não seria correto neste caso.

Reply:
boa!

Reply:
Perfeito, Tomas!
Só um ponto, consegue checar se a montagem do response esta correta? Estou recebendo essa erro na validação do schema:
![image](https://github.com/nic/no-redirect/assets/118825553/ba6b10a7-973b-4818-8e46-f14d2789da81)

Atte.,
José

Reply:
Bom dia, José. Qual ferramente você está utilizando para validar o arquivo? Utilizando o https://editor.swagger.io, o arquivo yml é processado sem erros, por isso a pergunta.
Obrigado.